realm/build.gradle

```diff
@@ -14,6 +15,7 @@ buildscript {
         classpath 'com.github.rholder:gradle-one-jar:1.0.4'
         classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.2'
         classpath 'com.netflix.nebula:gradle-extra-configurations-plugin:3.0.3'
+        classpath "io.realm:realm-transformer:${file('../version.txt').text.trim()}"
```

Comment:
Can't this be combined in some var and reused here and version declaration below?